src/App.test.tsx

```diff
@@ -1,12 +1,21 @@
-import { screen, render } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import App from './App';
+import render from './router/testRenderer';
 
-it('displays a +1 button', () => {
+it('App feature tour', async () => {
   render(<App />);
 
-  expect(
-    screen.getByRole('button', {
-      name: /\+1/i
-    })
-  ).toBeInTheDocument();
+  // now we expect to see all button options
+  screen.getByRole('button', {
+    name: /area/i
+  });
+  screen.getByRole('button', {
+    name: /tickets/i
+  });
+  screen.getByRole('button', {
+    name: /brand/i
+  });
+  screen.getByRole('button', {
+    name: /users/i
+  });
```

Comment:
I noticed your suggestion to individually check if the buttons render correctly on the screen. While it can offer more detailed test results, we should also keep test suite maintainability in mind.

Separate assertions for each button can be useful for inspecting individual behaviors but might lead to redundancy in larger suites. Let's weigh the trade-offs based on our project's specific needs and context. In the meantime, I still propose the assertions for **.toBeIntheDocument()** method on each button if you see the suggestion fit. Thanks for you great job, @Ralexs0096 !

```suggestion
  expect(screen.getByRole('button', { name: /area/i })).toBeInTheDocument();
  expect(screen.getByRole('button', { name: /tickets/i })).toBeInTheDocument();
  expect(screen.getByRole('button', { name: /brand/i })).toBeInTheDocument();
  expect(screen.getByRole('button', { name: /users/i })).toBeInTheDocument();
```

Reply:
Your suggestion is very good, and in fact, it is a good practice; the fact of not using it in this test is because the queryBy methods already assert the element, and if at some point the element changes either by text or role, this assertion should fail. So, doing it this way should work for the feature tests.